server/express.js

```diff
@@ -1,34 +1,76 @@
 const express = require('express')
+// const formData = require('express-form-data')
+const fetch = require('node-fetch')
+
 const app = express()
+
+// Parse URL-encoded bodies (as sent by HTML forms)
+app.use(express.urlencoded({ extended: true }))
+// Parse JSON bodies (as sent by API clients)
+app.use(express.json())
+
 const baseURL = 'https://www.metaweather.com/api'
-const fetch = require('node-fetch')
+
+const users = [
+  {
+    id: 1,
+    username: 'john',
+    password: '1234',
+    display_name: 'johnny',
+    is_admin: true
+  },
+  {
+    id: 2,
+    username: 'luci',
+    password: '1234',
+    display_name: 'lucian',
+    is_admin: false
+  }
+]
+
+// https://stackoverflow.com/questions/4295782/how-to-process-post-data-in-node-js
+app.post('/users', (req, res) => {
+  const foundUser = users.find(
+    (e) => e.username === req.body.username && e.password === req.body.password
+  )
+  if (!foundUser) {
+    res.status(404).send('User not found!')
```

Comment:
404 is Page not Found https://en.wikipedia.org/wiki/HTTP_404
 it's standard. Please don't use it just because something not found.

Reply:
why not return an empty object?

Reply:
A random string is not a standard thing (like null, undefined, {}, [], ''). This is not UI, this is API. So when u want to display some message like User not found, implement it on UI